pages/countries.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
 import Head from 'next/head'
 import NLink from 'next/link'
-
+import axios from 'axios'
 import styled from 'styled-components'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
 
 import {
   Flex, Box,
   Container,
   Link,
   Heading,
-  Text
 } from 'ooni-components'
+import { Text } from 'rebass'
```

Comment:
What is the reason to import `Text` from rebass instead of `ooni-components`?

Reply:
The one in `ooni-components` isn't configured for `fontWeight` prop. I didn't want to write custom StyledText just for that one part. The idea is to include that in the design-system and then just replace the import statement here.

I should have created an on `design-system` to reflect that.